webapp/portlet/src/main/webapp/vue-apps/version-history-drawer/components/VersionCard.vue

```diff
@@ -0,0 +1,223 @@
+<!--
+  This file is part of the Meeds project (https://meeds.io/).
+  Copyright (C) 2022 Meeds Association
+  contact@meeds.io
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 3 of the License, or (at your option) any later version.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <v-main>
+    <div
+      class="author-date-wrapper d-flex justify-space-between ">
+      <div class="version-author">
+        <span class="item-version border-radius primary px-2 font-weight-bold me-2 clickable">
+          V{{ versionObject.versionNumber }}
+        </span>
+        <span class="font-weight-bold text-truncate">{{ versionObject.authorFullName }}</span>
+      </div>
+      <div class="version-update-date">
+        <date-format
+          class="text-light-color text-truncate caption"
+          :value="versionDate"
+          :format="dateTimeFormat" />
+      </div>
+    </div>
+    <div class="description-restore-wrapper d-flex justify-space-between pt-2">
+      <br>
+      <div
+        v-if="enableEditDescription"
+        class="item-version-description">
+        <a
+          @click="showInput"
+          v-if="!versionObject.summary && descriptionInputHidden"
+          class="descriptionContent">
+          {{ $t('versionHistory.description.add') }}
+        </a>
+        <v-progress-circular
+          v-if="isUpdatingDescription"
+          :size="20"
+          color="primary"
+          indeterminate />
+        <v-tooltip
+          v-if="descriptionInputHidden"
```

Comment:
```suggestion
          v-else-if="descriptionInputHidden"
```
isn't this a `v-else-if` ?